tests/python/ExposureContrastTransformTest.py

```diff
@@ -0,0 +1,292 @@
+    TEST_GAMMA = 1.5
+    TEST_PIVOT = 0.3
+    TEST_LOGEXPOSURESTEP = 0.1
+    TEST_LOGMIDGRAY = 0.6
+    TEST_DIRECTION = OCIO.TRANSFORM_DIR_INVERSE
+
+    def setUp(self):
+        self.exp_tr = OCIO.ExposureContrastTransform()
+
+    def tearDown(self):
+        self.exp_tr = None
+
+    def test_style(self):
+        """
+        Test the setStyle() and getStyle() methods.
+        """
+
+        # Default initialized style is linear.
+        self.assertEqual(self.exp_tr.getStyle(), OCIO.EXPOSURE_CONTRAST_LINEAR)
+
+        for style in OCIO.ExposureContrastStyle.__members__.values():
+            self.exp_tr.setStyle(style)
+            self.assertEqual(self.exp_tr.getStyle(), style)
+
+    def test_exposure(self):
+        """
+        Test the setExposure() and getExposure() methods.
+        """
+
+        # Default initialized exposure is 0.
+        self.assertEqual(self.exp_tr.getExposure(), 0)
+
+        for exposure in self.TEST_VALUES:
+            self.exp_tr.setExposure(exposure)
+            self.assertEqual(self.exp_tr.getExposure(), exposure)
+
+        # Wrong type tests.
+        for invalid in (None, 'hello', [1, 2, 3]):
+            with self.assertRaises(TypeError):
+                self.exp_tr.setExposure(invalid)
+
+    def test_exposure_dynamic(self):
+        """
+        Test the isExposureDynamic() and makeExposureDynamic() methods.
+        """
+
+        # Default initialized property is False.
+        self.assertEqual(self.exp_tr.isExposureDynamic(), False)
+
+        self.exp_tr.makeExposureDynamic()
+        self.assertEqual(self.exp_tr.isExposureDynamic(), True)
+
+    def test_contrast(self):
+        """
+        Test the setContrast() and getContrast() methods.
+        """
+
+        # Default initialized contrast is 1.
+        self.assertEqual(self.exp_tr.getContrast(), 1)
+
+        for contrast in self.TEST_VALUES:
+            self.exp_tr.setContrast(contrast)
+            self.assertEqual(self.exp_tr.getContrast(), contrast)
+
+        # Wrong type tests.
+        for invalid in (None, 'hello', [1, 2, 3]):
+            with self.assertRaises(TypeError):
+                self.exp_tr.setContrast(invalid)
+
+    def test_contrast_dynamic(self):
+        """
+        Test the isContrastDynamic() and makeContrastDynamic() methods.
+        """
+
+        # Default initialized property is False.
+        self.assertEqual(self.exp_tr.isContrastDynamic(), False)
+
+        self.exp_tr.makeContrastDynamic()
+        self.assertEqual(self.exp_tr.isContrastDynamic(), True)
+
+    def test_gamma(self):
+        """
+        Test the setGamma() and getGamma() methods.
+        """
+
+        # Default initialized gamma is 1.
+        self.assertEqual(self.exp_tr.getGamma(), 1)
+
+        for gamma in self.TEST_VALUES:
+            self.exp_tr.setGamma(gamma)
+            self.assertEqual(self.exp_tr.getGamma(), gamma)
+
+        # Wrong type tests.
+        for invalid in (None, 'hello', [1, 2, 3]):
```

Comment:
Could define this list of invalid values once and reference it throughout the code.

Reply:
This is probably true of most of the test modules.

Reply:
Resolved with c60f458.